src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -2332,5 +2375,29 @@ private void AddMemberLookupSymbolsInfoInTypeParameter(LookupSymbolsInfo result,
                 AddMemberLookupSymbolsInfoWithoutInheritance(result, baseInterface, options, originalBinder, accessThroughType: type);
             }
         }
+
+        internal void AddImplicitExtensionMemberLookupSymbolsInfoForType(LookupSymbolsInfo result, TypeSymbol type, LookupOptions options, Binder originalBinder)
+        {
+            var accessThroughType = type;
```

Comment:
>var accessThroughType = type; [](http://example.com/codeflow?start=12&length=29)

Do we have tests demonstrating impact of this value?

Reply:
No, I don't think this will be observable until we have inheritance in extensions

Reply:
> No, I don't think this will be observable until we have inheritance in extensions

Are protected API declared in an extension accessible through extended type today? If they are not, then passing this ```accessThroughType``` value might make a difference.

Reply:
They are not. Added a Lookup test for this and there was no observable impact